### ISSUE TITLE
Missing "getting started" section

### ISSUE THREAD

**Issue by mourner** [assemble/verb#51]
The readme says "See the getting started section for details.", but the "getting started" link leads nowhere and there's no such section in the readme. :)

**Comment by jonschlinkert** [comment]
ha, well that's just great. A documentation generator with broken docs. thanks for letting me know!

**Comment by MattiSG** [comment]
+1

Also, I must say that after reading the README twice, I still don't really understand what `verb` is going to help me with… ok, it generates doc and it's magical and smells like chocolate, but what exactly do I write? Templates, fine, but what are they filled with? Which parts of the doc do I still need to write? Is `verb` some kind of replacement to JSdoc? To a README?…

**Comment by phun-ky** [comment]
@jonschlinkert I used this a couple of years ago, and the documentation for that was better, with howtos (add changelog) and the `.verbrc` file :smile: 

Did you do a full rewrite of the package?

**Comment by jonschlinkert** [comment]
> Did you do a full rewrite of the package?

yes, please see the `dev` branch. it's almost ready to merge to master. I'm working out some final CLI bugs
